Choreographer_30FPS/settings.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
For consolidating 3 project, I want to keep them separated because those demonstrates different features. Instead, I would make a bug for Android Studio importer to allow 'common' place for duplicated files.